index.js

```diff
@@ -7,25 +7,21 @@ import fs from "node:fs";
 import polka from "polka";
 import send from "@polka/send";
 import sirv from "sirv";
-import { WebSocketServer } from "ws";
 
 // Import Internal Dependencies
-import * as auth from "./src/authenticate.js";
 import * as orgCache from "./src/cache.js";
-import * as template from "./src/template.js";
-import DataFetcher from "./src/DataFetcher.class.js";
+import WSS from "./src/WebSocket.class.js";
 
 // CONSTANTS
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const kHttpPort = process.env.PORT || 1337;
-const kDataFetcher = new DataFetcher();
 
 fs.mkdirSync(orgCache.CACHE_PATH, {
   recursive: true
 });
 
 const httpServer = polka();
-const wsServer = new WebSocketServer({ port: 1338 });
+new WSS({ port: 1338 });
```

Comment:
```suggestion
new WSS({ port: kHttpPort });
```

Reply:
J'ai un `Error: listen EADDRINUSE: address already in use :::1337` vu que le serveur http utilise déjà ce port

Reply:
Ah oui, au temps pour moi